model/pcve.py

```diff
@@ -0,0 +1,55 @@
+from flask_restplus import fields, reqparse
+from app import server
+
+from src.types import FeedBackType, EventType
+from src.parse_datetime import to_date_str, from_date_str
+
+POST_PCVE = server.api.model('PCVE', {
+    'ecosystem': fields.String(description='Ecosystem'),
+    'repo_name': fields.String(description='Repository Name'),
+    'event_type': fields.String(description='Event Type'),
+    'status': fields.String(description='Status'),
+    'url': fields.Url(description='url'),
+    'id': fields.String(description='Event Id'),
+    'number': fields.Integer('PR/Issue number'),
+    'api_url': fields.Url(description='Github API'),
+    'created_at': fields.DateTime(dt_format='iso8601'),
+    'updated_at': fields.DateTime(dt_format='iso8601'),
+    'closed_at': fields.DateTime(dt_format='iso8601'),
+    'creator_name': fields.String,
+    'creator_url': fields.Url(description='Creator Url'),
+})
+
+class ISO8601Format(fields.Raw):
```

Comment:
Can we combine ISO8601Format class as well as methods inside parse_datetime file into single class

Reply:
parse_datetime is independent of `flask-restplus`. Moving `ISO8601Format` to parse_datetime adds additional dependency to testingb env which is not desirable. Also it is a simple class and it doesn't have any scope beyond the rest_model.py package.